### PR TITLE
[TR] PIM-4977: Re-allow full numeric entity codes

### PR DESCRIPTION
**Pull request by phaseinducer**
https://akeneo.atlassian.net/browse/PIM-4977

| Q                 | A
| ----------------- | ---
| Specs             | no regression
| Behats            | yes
| Blue CI           | yes
| Changelog updated | yes
| Review and 2 GTM  | yes

Reverts : https://github.com/akeneo/pim-community-dev/pull/3053

### CODE REVIEW

**Comment by phaseinducer** [comment]
Linked issues :
https://github.com/akeneo-labs/ExcelConnectorBundle/issues/21
https://github.com/akeneo/pim-community-dev/issues/3371

**Comment by jjanvier** [comment]
Fixes akeneo-labs/ExcelConnectorBundle#21

File: features/Context/catalog/footwear/attributes.yml [hunk 1]
```diff
@@ -333,3 +333,10 @@ attributes:
             reference_data_name: fabrics
         localizable:         true
         scopable:            true
+    123:
```
Comment (sumbobyboys) [inline features/Context/catalog/footwear/attributes.yml]: you should use 0123 because it was one of the problems we encounter :)

Comment (phaseinducer) [inline features/Context/catalog/footwear/attributes.yml]: 0123 will never be cast to integer so no difference with alphanumeric codes.

**Comment by nidup** [comment]
GTM

**Comment by jjanvier** [comment]
GTM

**Pull request merged**